app/graphql/types/workflow_execution_type.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Types
+  # WorkflowExecutions Type
+  class WorkflowExecutionType < Types::BaseObject
+    implements GraphQL::Types::Relay::Node
+    description 'A workflow execution'
+
+    field :blob_run_directory, String, null: true, description: 'WorkflowExecution blob run directory'
```

Comment:
**nitpick (blocking):** Can you remove this field, we don't want to expose this over the API.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/832/commits/6f79351d179108caa81252dc14a4e0032d61cd83